debian/postinst

```diff
@@ -11,16 +11,13 @@ ESM_APT_SOURCE_FILE_OLD="/etc/apt/sources.list.d/ubuntu-esm-trusty.list"
 ESM_APT_SOURCE_FILE="/etc/apt/sources.list.d/ubuntu-esm-infra-trusty.list"
 ESM_APT_PREF_FILE_OLD="/etc/apt/preferences.d/ubuntu-esm-trusty"
 ESM_APT_PREF_FILE="/etc/apt/preferences.d/ubuntu-esm-infra-trusty"
-UA_KEYRING_FILE="/usr/share/keyrings/ubuntu-advantage-keyring.gpg"
+UA_KEYRING_DIR="/usr/share/keyrings/"
```

Comment:
I'm not 100% sure that `/usr/share/keyrings` is the right place for these files any longer, as we're creating a bunch of them and the names are inscrutable for users. I don't know where _is_ the right place though, it just feels a little off.

Perhaps I'm just being too cautious though, and it's really fine! What do you think? Maybe @panlinux has an opinion?

Reply:
Agreed, I'm not certain either, that's why I threw this branch together to get your and @panlinux thoughts on this.

Reply:
I had a great response, then pressed ctrl-w by mistake.

Summarizing it, we used to have this:
```
keyrings/ubuntu-esm-v2-keyring.gpg
keyrings/ubuntu-fips-keyring.gpg
keyrings/ubuntu-esm-keyring.gpg
```
And now we have `keyrings/ua-$gibberish.gpg`. Why can't we have the names back, even if it's per ubuntu release, aka, a longer list?

Reply:
Per a side discussion on this, we decided we can drop the separate gpg files, so we will keep a single ringfile and avoid the potentially confusing ua-<key_id>.gpg files that mean nothing to an end user.